base/file.jl

```diff
@@ -638,18 +638,25 @@ function chmod(path::AbstractString, mode::Integer; recursive::Bool=false)
 end
 
 sentinel_code(i::Integer) = i
-sentinel_code(n::Nothing) = -1
-sentinel_code(n::Missing) = -2
-sentinel_code(a) = ArgumentError("$a is not an integer, nothing, or missing")
+sentinel_code(s::Symbol) =
+    if s == :preserve
+        -1
+    elseif s == :remove
+        -2
+    else
+        error("$s is not an integer, :preserve, or :remove")
+    end
 
 """
-    chown(path::AbstractString; owner = nothing, group = nothing)
+    chown(path::AbstractString; owner::Union{Symbol, Integer}, group::Union{Symbol, Integer})
 
 Change the owner and/or group of `path` to `owner` and/or `group`. If left as
-`nothing`, then the they won't be changed. If set to `missing`, they will be
+`:preserve`, then the they won't be changed. If set to `:remove`, they will be
 removed.
 """
-function chown(path::AbstractString; owner = nothing, group = nothing)
+function chown(path::AbstractString;
+    owner::Union{Symbol, Integer} = :preserve,
```

Comment:
Align arguments with `path`.

Reply:
That is not my preferred style.

Reply:
Funny, this file contains very few cases of wrapped arguments, and they use a right-alignment convention I hadn't seen anywhere before: https://github.com/JuliaLang/julia/pull/25694/files/23eb7a5484f9caed49b2a0fe37429f65378ad45d..21f3f5205fd4a09df7896e58ec33f2ff1dc4973c#diff-561359fb30664fed00e236179e3cead5R219

But in general I think the codebase aligns arguments, for example: https://github.com/bramtayl/julia/blob/21f3f5205fd4a09df7896e58ec33f2ff1dc4973c/base/iobuffer.jl#L24

Reply:
I like aligned arguments too but atom makes it difficult to do. Rstudio does it automatically. In general I follow the path of least resistance.

Reply:
I dislike aligned arguments since it means I may have to change many unrelated lines of code if the initial indent changes. Often, I find this just means that find/replace leaves behind a broken-formatted file. It also seems to have a tendency to look like everything is being crushed into the last 20-40 columns.

But this should probably have a double-indent, to help visually distinguish it from the body of the function.